src/dialog/dialog.service.ts

```diff
@@ -18,6 +18,11 @@ import { PlaceholderService } from "./../placeholder/placeholder.module";
  */
 @Injectable()
 export class DialogService {
+	/**
+	 * Used in `singletonClickListen`, don't count on its existence and values.
+	 */
+	static listeningForBodyClicks = false;
```

Comment:
Does `private static` or `static private` work in addition to the warning?

Reply:
Yep, I didn't think that was a thing. Today I learned...